vmdb/tools/fix_auth/fix_auth.rb

```diff
@@ -73,7 +73,7 @@ def fix_database_passwords
 
     def fix_database_yml
       FixDatabaseYml.file_name = "#{options[:root]}/config/database.yml"
-      FixDatabaseYml.run(run_options.merge(:hardcode => options[:password]))
+      FixDatabaseYml.run({:hardcode => options[:password]}.merge(run_options))
```

Comment:
Ahh ok.  Do we spit out an error and exit after trollop arg parsing if they specify both ?